.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Run Smoke Tests'
+on:
+  schedule:
+    - cron: '*/5 * * * *'
```

Comment:
Currently, the way GH implements this, the last person to *touch* the workflow will get notified if this fails.